client/src/aggregator.rs

```diff
@@ -37,3 +37,9 @@ pub struct NewAggregator {
     pub api_url: Url,
     pub bearer_token: String,
 }
+
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+pub struct CollectorAuthenticationToken {
+    pub r#type: String,
```

Comment:
Is this currently always expected to be `Bearer`? Could this be an internally-tagged enum?